includes/definitions/westmountainradio.yaml

```diff
@@ -0,0 +1,58 @@
+os: westmountainradio
+text: 'West Mountain RMCU'
+type: power
+mib_dir: westmountainradio
+icon: westmountainradio
+nobulk: true
+over:
+    - { graph: device_current, text: Current }
+    - { graph: device_voltage, text: Voltage }
+    - { graph: device_state, text: 'Power Status' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.15117.1
+poller_modules:
+    applications: false
```

Comment:
Please don't disable modules if not needed due to buggy devices

Reply:
Ok, left only enabled modules - noting that this increases polling & discovery time substantially

Reply:
Please Remove all, it's really only meant for buggy devices